src/pamiq_core/torch/model.py

```diff
@@ -114,6 +114,7 @@ def __init__(
         device: torch.device | str | None = None,
         dtype: torch.dtype | None = None,
         inference_procedure: InferenceProcedureCallable = default_infer_procedure,
+        parameter_file: str | Path | None = None,
```

Comment:
これ`pretrained_file`とか`pretrained_model`とかみたいに、事前学習済みモデルのロードであることを明記したさある
checkpointのロードと混同しうるのを避けたい　そっちは`load_state`でやりたいはずだろうし
```suggestion
        pretrained_file: str | Path | None = None,
```

Reply:
`pretrained`かどうかは文脈依存だと思う。これは純粋に `parameter`という意味が大事

Reply:
`save_state`, `load_state`はpamiq-coreのフレームワークに従った状態保存、読み込みなので、ユーザーが任意に学習済みモデルを使用可能なAPIを残しておく必要がある。